README.md

```diff
@@ -58,4 +58,33 @@ This repository is specifically laid out to make it easy to add new features and
 
 A new version branch would only ever be created if a significant change is being made which would have a major impact on consumers of the script plugin. In most cases, this is likely because the build scripts that include the script plugin would need to change themselves, but this would also apply if there would be a fundamental change to the artifacts being produced that would need consideration before updating an app's build script.
 
-Once a new version exists, only bug fixes should ever be added to older versions. These bug fixes should be merged into newer versions if they are still applicable.
\ No newline at end of file
+Once a new version exists, only bug fixes should ever be added to older versions. These bug fixes should be merged into newer versions if they are still applicable.
+
+## Publishing
+
+If your project will be publishing an AAR, you will also need to apply the [ttm_module_publish gradle file](ttm_module_publish.gradle):
+
+```gradle
+apply from: 'https://www.github.com/geotracsystems/ttm-mep-build-scripts/raw/master/ttm_module_publish.gradle'
+```
+
+Additionally, you will need to set a `version` property in the root project:
+
+```gradle
+version="1.0"
```

Comment:
Where do I have to define this? Just anywhere in the root `build.gradle` file? It doesn't need to be in an ext block or configurations.all or anything like that? It would be useful to show a little more context around this to help someone figure out where this actually goes.

Reply:
I will update the readme. It has to be at the root level of the root gradle file